policy/aggregation_type_options.go

```diff
@@ -283,6 +289,29 @@ func (o *options) initPools() {
 	o.quantilesPool.Init()
 }
 
+func (o *options) Validate() error {
+	if err := o.validateUniqSuffixForMetricType(o.counterSuffixes, metric.CounterType); err != nil {
```

Comment:
nit: Perhaps `ensureUniqueSuffixesFor(...)` is better?

Reply:
sounds good, will do ```ensureUniqueSuffix```